vue/src/store/modules/search.js

```diff
@@ -23,9 +24,16 @@ export default {
         day: '2020-05-16'
       }
     ]
+  },
+  mutations: {
+    setSearch (state, payload) {
+      state.items = payload
+    }
+  },
+  actions: {
+    fetchSearch ({ commit }) {
```

Comment:
```suggestion
    fetchItems ({ commit }) {
```
とかが適当かな。メソッド名動詞＋動詞になっている